ImageData/Hdf5Loader.cc

```diff
@@ -200,206 +200,172 @@ bool Hdf5Loader::UseRegionSpectralData(const std::shared_ptr<casacore::ArrayLatt
     return true;
 }
 
-bool Hdf5Loader::GetRegionSpectralData(int region_id, int config_stokes, int profile_stokes,
-    const std::shared_ptr<casacore::ArrayLattice<casacore::Bool>> mask, IPos origin, std::mutex& image_mutex,
-    const std::function<void(std::map<CARTA::StatsType, std::vector<double>>*, float)>& partial_results_callback) {
-    // config_stokes is the stokes value in the spectral config, could be -1 for CURRENT_STOKES; used to retrieve config from region
-    // profile_stokes is the stokes index to use for slicing image data
+bool Hdf5Loader::GetRegionSpectralData(int region_id, int stokes, const casacore::Array<casacore::Bool>& mask,
+    const casacore::IPosition& origin, std::mutex& image_mutex, std::map<CARTA::StatsType, std::vector<double>>& results, float& progress) {
```

Comment:
Since I am familiar with casacore, I think "casacore::IPosition" when I develop code.  This is defined in FileLoader.h and only used in Hdf5Loader, so you have to search the code to see what this thing is.  So my personal preference is not to use aliases, but I will change HdfLoader to be consistent.